### ISSUE TITLE
[New Rule]: rewrite dict.get(key, None) as dict.get(key)

### ISSUE THREAD

**Issue by janosh** [MartinThoma/flake8-simplify#171]
## Explanation

`dict.get(key)` returns `None` by default if `key` is missing.

## Example

```python
# Bad
dict.get(key, None)

# Good
dict.get(key)
```

## Related

Adding this rule should be coordinated with https://github.com/charliermarsh/ruff/issues/3546 for consistent naming.

**Comment by kyoto7250** [comment]
I am interested in this issue. Would you mind if I tackle this issue?

In making this rule, what will the `SIM` number be?

**Comment by MartinThoma** [comment]
Go ahead :-) 🙏

The number should be SIM910
